src/components/selfCheckQuestion/index.tsx

```diff
@@ -0,0 +1,450 @@
+                                    variant="ghost"
+                                    onClick={() => onAddOption(questionId)}
+                                >
+                                    +Add Option
+                                </Button>
+                            </Stack>
+                        </Flex>
+                    )}
+                    {type == "multi_select" && (
+                        <Flex width={"full"}>
+                            <Stack minWidth={"50%"} spacing={3}>
+                                {(options ?? []).map((option, index) => (
+                                    <InputGroup key={option + questionId}>
+                                        <InputLeftElement
+                                            pointerEvents="none"
+                                            children={
+                                                <Square
+                                                    size="15px"
+                                                    borderWidth="2px"
+                                                    borderColor="black"
+                                                    borderRadius="sm"
+                                                ></Square>
+                                            }
+                                        />
+                                        <Input
+                                            variant="flushed"
+                                            placeholder={`Option ${index + 1}`}
+                                            mr={6}
+                                            isTruncated
+                                        />
+                                        <InputRightElement width="4.5rem">
+                                            <CloseButton
+                                                name={questionId}
+                                                onClick={() =>
+                                                    onRemoveOption(
+                                                        questionId,
+                                                        index,
+                                                    )
+                                                }
+                                            />
+                                        </InputRightElement>
+                                    </InputGroup>
+                                ))}
+                                <Button
+                                    maxWidth={"120"}
+                                    variant="ghost"
+                                    onClick={() => onAddOption(questionId)}
+                                >
+                                    +Add Option
+                                </Button>
+                            </Stack>
+                        </Flex>
+                    )}
+                    {type == "short_answer" && (
+                        <Textarea
```

Comment:
should we impose a word limit here since it's short answer

Reply:
Also we need an onchange here

Reply:
> should we impose a word limit here since it's short answer

this would be good idea - let's discuss with the PM/designers

as for the onchange i don't know if we really need that - since the textarea is more so for the users to fill out right? there's no default values afaik, we should also confirm this with the product team

Reply:
I think the word limit should only be on the client side, in fact, after talking to the product team I don't think the short answer on the admin side should even be an editable textbox, and is rather just there to show what the textbox would look like for the end user